lib/benchee/formatters/console.ex

```diff
@@ -62,27 +62,52 @@ defmodule Benchee.Formatters.Console do
     max_label_width + 1
   end
 
-  defp job_reports(jobs, label_width) do
-    Enum.map(jobs, fn(job) -> format_job job, label_width end)
+  defp job_reports(jobs, units, label_width) do
+    Enum.map(jobs, fn(job) -> format_job job, units, label_width end)
+  end
+
+  defp units(jobs) do
+    # Produces a map like
+    #   %{run_time: [12345, 15431, 13222], ips: [1, 2, 3]}
+    collected_values = jobs
+    |> Enum.flat_map(fn({_name, job}) -> Map.to_list(job) end)
```

Comment:
hm code formatting wise I'm usually a fan (but not yet decided on which one) of:

```elixir
mine = value
       |> pipe_stuff

# or

mine =
  value
  |> pipe_stuff
```

makes the start of the pipe more visible I think and more distinguished. What do you think?